jobs/src/main/scala/dcos/metronome/model/JobSpec.scala

```diff
@@ -29,14 +33,36 @@ case class JobSpec(
 
 object JobSpec {
   val DefaultDescription = None
+  val DefaultDependencies = Option.empty[Seq[JobId]]
```

Comment:
The job ids would require to join each time. It's simpler to deserialize though.